lib/export/export_excel.rb

```diff
@@ -119,9 +119,14 @@ def filter_with_permission_and_detail_level(sample)
       # return all data if sample in own collection
       if sample['shared_sync'] == 'f' || sample['shared_sync'] == false
         headers = @headers
+        comparison_values = ['melting pt', 'boiling pt']
```

Comment:
the name comparison_values is for me a little bit misleading. Maybe interval_values or intervaluable_values is clearer.